Tests/MarkdownTests/Infrastructure/SourceLoacationTests.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+@testable import Markdown
+import XCTest
+
+class SourceLoacationTests: XCTestCase {
+    func testNonAsciiCharacterColumn() throws {
```

Comment:
It seems like the column values are retrieved from cmark (`cmark_node_get_start_column`) and thus I'm not convinced the cmark behaviour should be tested here. I don't think we need this test.

Reply:
I think we are not mainly testing the cmark behavior. We are convincing ourselves that the doc of SourceLocation - "The number of bytes in UTF-8 encoding" will not break unintentionally. Also, the cmark test does not cover Emoji and CJK characters. And it will be fine if we add some tests here.

Reply:
Gotcha. So this is more of an integration test. I think this is beneficial then, as you explained, to make sure the doc comment remains valid. Can we remove the print statements?

Reply:
OK